### ISSUE TITLE
Fix warning about text domain loading too early

### ISSUE THREAD

**Issue by knice** [ucsc/theme-ucsc#387]
https://make.wordpress.org/core/2024/10/21/i18n-improvements-6-7/

> Notice: Function `_load_textdomain_just_in_time` was called incorrectly. Translation loading for the `ucsc-2022` domain was triggered too early. This is usually an indicator for some code in the plugin or theme running too early. Translations should be loaded at the `init` action or later. Please see [Debugging in WordPress](https://developer.wordpress.org/advanced-administration/debug/debug-wordpress/) for more information. (This message was added in version 6.7.0.)
